Lib/fontTools/otlLib/builder.py

```diff
@@ -2734,17 +2734,17 @@ def buildStatTable(ttFont, axes, locations=None, elidedFallbackName=2):
     ttFont["STAT"] = ttLib.newTable("STAT")
     statTable = ttFont["STAT"].table = ot.STAT()
     nameTable = ttFont["name"]
-    statTable.ElidedFallbackNameID = _addName(nameTable, elidedFallbackName)
+    statTable.ElidedFallbackNameID = _addName(nameTable, elidedFallbackName, windows=windowsNames, mac=macNames)
 
     # 'locations' contains data for AxisValue Format 4
-    axisRecords, axisValues = _buildAxisRecords(axes, nameTable)
+    axisRecords, axisValues = _buildAxisRecords(axes, nameTable, windowsNames=windowsNames, macNames=macNames)
     if not locations:
         statTable.Version = 0x00010001
     else:
         # We'll be adding Format 4 AxisValue records, which
         # requires a higher table version
         statTable.Version = 0x00010002
-        multiAxisValues = _buildAxisValuesFormat4(locations, axes, nameTable)
+        multiAxisValues = _buildAxisValuesFormat4(locations, axes, nameTable, windowsNames=windowsNames, macNames=macNames)
```

Comment:
ok since we are nitpicking, this line is way too long, we should stay within 88 max characters. Because this `otlLib.builder` module was already passed through the `black` autoformatter once (according to git log), could you please run again the `black` command, on this file only?
```
$ pip install black
$ black Lib/fontTools/otlLib/builder.py
```

Reply:
Also blacken `Tests/otlLib/builder_test.py` please, thanks

Reply:
Uh, these commands changed a lot. If this is what you want, please see the latest code commits @anthrotype @justvanrossum